src/WordPress/WordPressAPI.php

```diff
@@ -8,6 +8,7 @@
 class WordPressAPI implements IntegrationAPIInterface
 {
     private $dataStore;
+    private static $domainName;
```

Comment:
`static $domainName` would not be garbage collected when a regular request is made. When an action which calls a hook would need to use the `getDomainList`, `domainName` would still be set and we won't have to make another API call.
```
    public function getDomainList($userId = null)
    {
        return array(self::$domainName);
    }
```